requirements.txt

```
@@ -1,35 +1,22 @@
-alembic==1.5.4
-attrs==20.3.0
-autopep8==1.5.5
-blinker==1.4
-certifi==2020.12.5
-chardet==4.0.0
-click==7.1.2
-coverage==6.3.3
-Flask==1.1.2
-Flask-Cors==3.0.10
-Flask-Migrate==2.6.0
-Flask-SQLAlchemy==2.4.4
-gunicorn==20.1.0
-idna==2.10
-iniconfig==1.1.1
-itsdangerous==1.1.0
-Jinja2==2.11.3
-Mako==1.1.4
-MarkupSafe==1.1.1
-packaging==20.9
-pluggy==0.13.1
-py==1.10.0
-pycodestyle==2.6.0
-pyparsing==2.4.7
-pytest==6.2.3
-pytest-cov==2.12.1
-python-dateutil==2.8.1
-python-dotenv==0.15.0
-python-editor==1.0.4
-requests==2.25.1
-six==1.15.0
-SQLAlchemy==1.3.23
-toml==0.10.2
-urllib3==1.26.5
-Werkzeug==1.0.1
+alembic==1.13.1
+blinker==1.7.0
+click==8.1.7
+Flask==3.0.2
+Flask-Cors==5.0.0
+Flask-Migrate==4.0.5
+Flask-SQLAlchemy==3.1.1
+greenlet==3.0.3
+gunicorn==21.2.0
+iniconfig==2.0.0
+itsdangerous==2.1.2
+Jinja2==3.1.3
+Mako==1.3.2
+MarkupSafe==2.1.5
+packaging==23.2
+pluggy==1.4.0
+psycopg2-binary==2.9.9
+pytest==8.0.0
+python-dotenv==1.0.1
+SQLAlchemy==2.0.25
+typing_extensions==4.9.0
+Werkzeug==3.0.1
```

It looks like we're missing the `requests` package & its dependencies in requirements.txt. When I tried to run the server I got the error:
```py
  File "/Users/kelseysteven/Documents/curriculum_changes/weather-report-proxy-server/app/routes.py", line 5, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

Once I did `pip install requests`, I was able hit both the endpoints as expected.

```suggestion
alembic==1.13.1
blinker==1.7.0
certifi==2024.8.30
charset-normalizer==3.4.0
click==8.1.7
Flask==3.0.2
Flask-Cors==5.0.0
Flask-Migrate==4.0.5
Flask-SQLAlchemy==3.1.1
greenlet==3.0.3
gunicorn==21.2.0
idna==3.10
iniconfig==2.0.0
itsdangerous==2.1.2
Jinja2==3.1.3
Mako==1.3.2
MarkupSafe==2.1.5
packaging==23.2
pluggy==1.4.0
psycopg2-binary==2.9.9
pytest==8.0.0
python-dotenv==1.0.1
requests==2.32.3
SQLAlchemy==2.0.25
typing_extensions==4.9.0
urllib3==2.2.3
Werkzeug==3.0.1
```